frontend/packages/api_client/src/apis/V2Api.ts

```diff
@@ -575,6 +583,52 @@ export class V2Api extends runtime.BaseAPI {
         return await response.value();
     }
 
+    /**
+     * This is a custom handler to partially update theme fields.
+     * Partial update for theme configuration
+     */
+    async instancesOpenedxConfigThemeConfigRaw(requestParameters: InstancesOpenedxConfigThemeConfigRequest): Promise<runtime.ApiResponse<ThemeSchema>> {
+        if (requestParameters.id === null || requestParameters.id === undefined) {
+            throw new runtime.RequiredError('id','Required parameter requestParameters.id was null or undefined when calling instancesOpenedxConfigThemeConfig.');
+        }
+
+        if (requestParameters.data === null || requestParameters.data === undefined) {
+            throw new runtime.RequiredError('data','Required parameter requestParameters.data was null or undefined when calling instancesOpenedxConfigThemeConfig.');
+        }
+
+        const queryParameters: runtime.HTTPQuery = {};
+
+        const headerParameters: runtime.HTTPHeaders = {};
+
+        headerParameters['Content-Type'] = 'application/json';
+
+        if (this.configuration && this.configuration.apiKey) {
+            headerParameters["Authorization"] = this.configuration.apiKey("Authorization"); // api_key authentication
+        }
+
+        if (this.configuration && (this.configuration.username !== undefined || this.configuration.password !== undefined)) {
+            headerParameters["Authorization"] = "Basic " + btoa(this.configuration.username + ":" + this.configuration.password);
+        }
+        const response = await this.request({
+            path: `/v2/instances/openedx_config/{id}/theme_config/`.replace(`{${"id"}}`, encodeURIComponent(String(requestParameters.id))),
```

Comment:
@giovannicimolin, can the URL, at least the base part `/v2/instances/openedx_config/` if not the whole URL with the placeholder, be defined somewhere as a constant outside of the usage so that it is easier to check and change in the future?

Reply:
@lgp171188 This is auto-generated code from the Swagger spec, I think that this is pretty much hardcoded into how the generator works, so we'll not going to be able to change it (unless we do a manual patch, but we don't do that in any auto-generated code).